scripts/build

```
@@ -137,6 +164,10 @@ TARGET_MESON_OPTS="--prefix=/usr \
                    --libexecdir=/usr/lib \
                    --localstatedir=/var \
                    --buildtype=${MESON_BUILD_TYPE}"
+if [ ${BUILD_WITH_DEBUG} != yes ] && flag_enabled "strip" "yes"; then
```

Tiny nit, just to be consistent with all other instances:
```suggestion
if [ "${BUILD_WITH_DEBUG}" != "yes" ] && flag_enabled "strip" "yes"; then
```

---

Copy'n'paste mistake XD